chain/validation/state.go

```diff
@@ -293,6 +302,8 @@ func fromSingletonAddress(addr vstate.SingletonActorID) vstate.Address {
 		return vstate.Address(actors.StorageMarketAddress.Bytes())
 	case vstate.BurntFundsAddress:
 		return vstate.Address(actors.BurntFundsAddress.Bytes())
+	case vstate.StoragePowerAddress:
```

Comment:
We should figure out a way to not have this switch here, and just cast one address to the other

Reply:
Agreed, currently these are just [constants](https://github.com/filecoin-project/chain-validation/blob/master/pkg/state/actors.go#L3) since their addresses are not yet defined across all implementations. I see this as a temporary solution.